avpipe.go

```diff
@@ -1283,15 +1282,15 @@ func getCParams(params *XcParams) (*C.xcparams_t, error) {
 		cparams.listen = C.int(1)
 	}
 
-	if params.NumAudio > MaxAudioMux {
-		return nil, fmt.Errorf("Invalid number of audio streams NumAudio=%d", params.NumAudio)
+	if int32(len(params.AudioIndex)) > MaxAudioMux {
+		return nil, fmt.Errorf("Invalid number of audio streams NumAudio=%d", len(params.AudioIndex))
```

Comment:
I think we can remove MaxAudioMux - can we?

Reply:
The purpose of MaxAudioMux is to avoid memory fragmentation. It lets us allocate XcParams in one chunk of memory.

Reply:
Ok